common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -290,6 +299,91 @@ def _check_scores_and_page_text(self, problem_scores, section_score, text):
         self.assertEqual(self._get_section_score(), section_score)
         self.assertTrue(self.progress_page.text_on_page(text))
 
+    def _check_tick_text(self, index, sr_text, label, label_hidden=True):
+        """
+        Check the label and sr text for a horizontal (X-axis) tick.
+        """
+        self.assertEqual(sr_text, self.progress_page.x_tick_sr_text(index))
+        self.assertEqual([label, 'true' if label_hidden else None], self.progress_page.x_tick_label(index))
+
+    def test_axis_a11y(self):
+        """
+        Tests that the progress chart axis have appropriate a11y (screenreader) markup.
+        """
+        with self._logged_in_session():
+            self.courseware_page.visit()
+            self._answer_problem_correctly()
+            self.courseware_page.click_next_button_on_top()
+            self._answer_problem_correctly()
+            self.progress_page.visit()
+
+            # Verify that y-Axis labels are aria-hidden
+            self.assertEqual(['100%', 'true'], self.progress_page.y_tick_label(0))
+            self.assertEqual(['0%', 'true'], self.progress_page.y_tick_label(1))
+            self.assertEqual(['Pass 50%', 'true'], self.progress_page.y_tick_label(2))
+
+            # Verify x-Axis labels and sr-text
+            self._check_tick_text(0, [u'Homework 1 - Test Subsection 1 - 50% (1/2)'], u'HW 01')
+
+            # Homeworks 2-10 are checked in the for loop below.
+
+            self._check_tick_text(
+                10,
+                [u'Homework 11 Unreleased - 0% (?/?)', u'The lowest 2 Homework scores are dropped.'],
```

Comment:
These tooltips are a bit funky because no actual content exists for these assignments.